middlewares/verifyToken.js

```diff
@@ -0,0 +1,18 @@
+function verifyToken(req, res, next) {
+  const bearerHeader = req.headers.authorization;
+  if (bearerHeader !== 'undefined') {
```

Comment:
`undefined` is not a string

Reply:
`if (bearerHeader)` is enough

Reply:
fixed

Reply:
Have not pused yet, as `verifyToken` logic has not been fixed